test/index.js

```diff
@@ -115,6 +119,16 @@ test ('Json middleware', () => {
   eq (mockNext.args, []);
 });
 
+test ('Render middleware', () => {
+  const mock = middleware (_ => resolve (Render (200) ('index') ({user: 'hello'})));
+  const mockRes = {status: methodSpy (), render: methodSpy ()};
+  const mockNext = sinon.spy ();
+
+  mock ({}, mockRes, mockNext);
+
+  eq (mockRes.status.args, [[200]]);
```

Comment:
Could we add an assertion to check if `render` got called correctly? eg:

```suggestion
  eq (mockRes.status.args, [[200]]);
  eq (mockRes.render.args, [['index', {user: 'hello'}]]);
```

Reply:
Yup, this seemed to work. Committing.